tests/mcp/cloudrun/main.tf

```diff
@@ -36,15 +36,15 @@ output test_traffic_config {
   value = data.external.test_traffic_with_revision.result
 }
 
-//tests traffic block with no settings, run `terraform apply -var 'gcp_cloudrun_yml=resrouces/empty_traffic.yml'
-data external test_traffic_empty{
-  query = local.cloudrun_traffic["default"] == [] ? {} : {"format" = "not empty list"}
-  program = ["python", "${path.module}/test_traffic_empty.py"]
-}
-
-output test_traffic_empty {
-  value = data.external.test_traffic_empty.result
-}
+////tests traffic block with no settings, run `terraform apply -var 'gcp_cloudrun_yml=resources/empty_traffic.yml'
```

Comment:
This test was commented because it was a test that needed to use `terraform apply -var 'gcp_cloudrun_yml=resources/empty_traffic.yml'` 
Because of this, the test would always fail in the automated unit testing

Reply:
If we're leaving it in but commented for some reason. Add a message describing why it is commented

Reply:
Realised can have different service in the yaml instead of a different file. Test was originally written before had updated it to allow for multiple services